_api-reference/index-apis/shrink-index.md

```diff
@@ -56,10 +56,21 @@ timeout | Time | How long to wait for the request to return a response. Default
 You can use the request body to configure some index settings for the target index. All fields are optional.
 
 Field | Type | Description
+:--- | :--- | :---
 alias | Object | Sets an alias for the target index. Can have the fields `filter`, `index_routing`, `is_hidden`, `is_write_index`, `routing`, and `search_routing`. See [Index Aliases]({{site.url}}{{site.baseurl}}/api-reference/alias/#request-body).
 settings | Object | Index settings you can apply to your target index. See [Index Settings]({{site.url}}{{site.baseurl}}/api-reference/index-apis/create-index/#index-settings).
+[max_shard_size](#the-max_shard_size-parameter) | Bytes | Specifies the maximum size of a primary shard in the target index. Because `max_shard_size` conflicts with the `index.number_of_shards` setting, you cannot set both of them at the same time. 
+
+### The `max_shard_size` parameter
+
+ The `max_shard_size` parameter specifies the maximum size of a primary shard in the target index. OpenSearch uses `max_shard_size` and the total storage for all primary shards in the source index to calculate the number of primary shards and their size for the target index. 
+ 
+ The primary shard count of the target index is the lowest factor of the source index's primary shard count, for which the shard size does not exceed `max_shard_size`. Consider the following example. Let's say the source index has 8 primary shards and they occupy a total of 400 GB of storage. If `max_shard_size` is equal to 150 GB, OpenSearch calculates the number of primary shards in the target index using the following algorithm:
```

Comment:
1. Are these paragraphs under the heading indented? Extra space here?
2. "Smallest" factor sounds more natural to me than "lowest" factor. But this might be a convention in mathematics and widely accepted. Just asking.
3. "... of the source index's primary shard count, whose shard size should not [will not?] exceed `max_shard_size`." Does that mess up the meaning?

Reply:
1. Removed extra space, thank you. Does not affect the rendering, but still a good call.
2. Agreed. Smallest is better.
3. I feel like "whose" makes it less clear because it's not clear what it refers to.